.github/workflows/format.yml

```diff
@@ -0,0 +1,30 @@
+name: Format suggestions
+
+on:
+  pull_request:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@19bd4db8c427e07d56602f1c432e1a26a55d6735 # v1.8.2
+        with:
+          version: 1
+      - shell: julia --color=yes {0}
+      - run: |
+          using Pkg
+          Pkg.add(PackageSpec(name="JuliaFormatter", version="1.0.10"))
+          using JuliaFormatter
+          format(".", MinimalStyle(); normalize_line_endings = "unix")
+      - uses: reviewdog/action-suggester@v1
```

Comment:
Are the annotations left by `reviewdog/action-suggester` really helpful ? IMO, they just pollute the `Files changed` view by adding tons of annotations. I know they can be disabled (manual intervention needed - cumbersome), but they tend to distract attention. I think that the failing `JuliaFormatter` is enough.

Reply:
PRs with many comments, like for example https://github.com/JuliaLang/julia/pull/43990, would become even more unwieldy though.

Reply:
which is why I always wanted it to be directly pushing to fork branch

Reply:
I mean idk, it seems people think nothing is better than anything so maybe I will just drop this